smartsim/database/orchestrator.py

```diff
@@ -165,6 +165,19 @@ def check_cluster_status(self):  # cov-wlm
         if trials == 0:
             raise SmartSimError("Cluster setup could not be verified")
 
+    def _get_address(self):
+        """Return database addresses
+
+        :return: addresses
+        :rtype: list[str]
+        """
+        if not self._hosts:
+            raise SmartSimError()
```

Comment:
I like this conditional, we should probably have an error message though so the user isn't left without knowing what happened. maybe something like, "Could not find database address".